### PR TITLE
Deprecations and removals for Chrome 75.

### PR DESCRIPTION
**Pull request by jpmedley**
**Target Live Date:** 2019-05-02

- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2019/05/chrome-75-deps-rems.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: A round up of the deprecations and removals in Chrome 75 to help you plan. #}
+
+{% include "web/updates/_shared/see-all-dep-rem.html" %}
+
+# Deprecations and removals in Chrome 75 {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+## Remove overflow: -webkit-paged-x and overflow: -webkit-paged-y
```
Comment (petele) [inline src/content/en/updates/2019/05/chrome-75-deps-rems.md]: backtick overflow

File: src/content/en/updates/2019/05/chrome-75-deps-rems.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+These are old webkit-specific properties that allowed developers to fragment
+content over columns inside a scrollable region. They are now removed.
+
+Practically speaking aren't used. Most developers use it accidentally, and
```
Comment (petele) [inline src/content/en/updates/2019/05/chrome-75-deps-rems.md]: "Practically speaking aren't used." sentence fragment

Comment (petele) [inline src/content/en/updates/2019/05/chrome-75-deps-rems.md]: "Most developers use it accidentally" what is "it"? And you've switched from plural to singular

File: src/content/en/updates/2019/05/chrome-75-deps-rems.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+typically when it is used, this property forces a new formatting context
+similar to setting "overflow: hidden".
```
Comment (petele) [inline src/content/en/updates/2019/05/chrome-75-deps-rems.md]: Backtick `overflow: hidden`

**Comment by jpmedley** [comment]
PTAL

**Pull request merged**